modules/src/clients/host_functions.rs

```diff
@@ -1,10 +1,18 @@
 use crate::core::ics02_client::error::Error;
 use crate::prelude::*;
+use beefy_client_primitives::HostFunctions as BeefyHostFunctions;
 use core::marker::PhantomData;
+#[cfg(any(test, feature = "ics13_near"))]
+use near_primitives_wasm::HostFunctions as MaybeNearHostFunctions;
+
+#[cfg(not(any(test, feature = "ics13_near")))]
+trait MaybeNearHostFunctions {}
 
 /// This trait captures all the functions that the host chain should provide for
 /// crypto operations.
-pub trait HostFunctionsProvider: Clone + Send + Sync + Default {
+pub trait HostFunctionsProvider:
+    Clone + Send + Sync + Default + BeefyHostFunctions + MaybeNearHostFunctions
```

Comment:
```suggestion
    Clone + Send + Sync + Default 
```

see: https://github.com/ComposableFi/ibc-rs/blob/55fc39714c150a11c1d13a93dd6de60eb051d47c/modules/src/clients/host_functions.rs#L77-L81